arangod/RestHandler/RestCursorHandler.cpp

```diff
@@ -659,6 +678,73 @@ RestStatus RestCursorHandler::createQueryCursor() {
   return registerQueryOrCursor(body);
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// @brief shows the batch given by <batch-id> if it's the last cached batch
+/// response on a retry, and does't advance cursor
+////////////////////////////////////////////////////////////////////////////////
+
+RestStatus RestCursorHandler::showLastBatchRetry() {
```

Comment:
I find the method name hard to understand. `showLastBatch()` I do understand, but the extra `Retry` at the end puzzles me a bit.